File: src/com/github/legendofmcpe/pocketbot/packet/raknet/OpenConnectionReply1.java
package com.github.legendofmcpe.pocketbot.packet.raknet;

import com.github.legendofmcpe.pocketbot.PocketBot;

public class OpenConnectionReply1 extends ReceivedRaknetPacket{
	private short mtu;

	public byte getPid(){
		return OPEN_CONNECTION_REPLY_1;
	}

	@Override
	protected void decode(PocketBot bot){
		bb().get(new byte[MAGIC_BYTES.length]); // magic
		bb().getLong(); // server ID
		bb().get(); // security
		mtu = bb().getShort();
	}

	public short getMtuCnt(){
		return mtu;
	}
}

File: src/LegendOfMCPE/PocketBot/network/raknet/OpenConnectionReply1.php
<?php

namespace LegendOfMCPE\PocketBot\network\raknet;

class OpenConnectionReply1 extends RaknetPacket{
	/** @var string */
	private $magic;
	/** @var int */
	private $serverId;
	/** @var int */
	private $security;
	/** @var int */
	private $mtu;
	public function getPid(){
		return self::OPEN_CONNECTION_REPLY_1;
	}
	public function decode(){
		$this->magic = $this->read(strlen(self::MAGIC));
		$this->serverId = $this->readLong();
		$this->security = $this->readChar();
		$this->mtu = $this->readShort(false);
	}
	public function getMagic(){
		return $this->magic;
	}
	public function getServerId(){
		return $this->serverId;
	}
	public function getSecurity(){
		return $this->security;
	}
	public function getMtu(){
		return $this->mtu;
	}
}
